docs/guidelines.md

```diff
@@ -0,0 +1,195 @@
+# Actions Guidelines
+
+This document outlines a shared set of guidelines for creating and maintaining GitHub Actions files for Hemi Labs.
+
+---
+
+## 1. Reusable actions and workflows
+
+The [`hemilabs/actions`](https://github.com/hemilabs/actions/) repository is the primary repository for GitHub Actions
+and workflows.
+
+- Workflows or actions used in multiple repositories should be placed here, unless there's a specific reason not
+  to (e.g., internal, or environment-specific logic).
+- Aim to make actions and workflows configurable and well-documented to allow reuse.
+- Breaking changes to shared actions or workflows should be discussed prior to being made to avoid unexpected issues in
+  consuming projects.
+
+### 1.1. Versioning
+
+The `hemilabs/actions` repository uses tagged releases, following Semantic Versioning. Consumers of actions or workflows
+from `hemilabs/actions` should pin the action/workflows to a `vX.X` or `vX.X.X` tag to avoid unexpected or unreviewed changes.
+
+When breaking changes may need to be made to a certain action or workflow, an appropriate version bump must be made to
+avoid disrupting existing consumers.
+
+### 1.2. Reviews
+
+All changes made to the `hemilabs/actions` repository should be reviewed by two team members from the team that owns
+the updated files. When multiple teams are owners, at least one reviewer from each team should approve the changes to
+ensure alignment and avoid surprises.
+
+Additionally, for security purposes, all changes require a sign-off from a member of the `@hemilabs/devops` or
+`@hemilabs/secops` teams.
+
+### 1.3. Ownership
+
+Each team should be responsible for maintaining the workflows or actions they contribute to this repository.
+When contributing workflows or actions, add your team as a [code owner](../.github/CODEOWNERS), unless agreed upon. This
+helps ensure appropriate review coverage and accountability.
+
+## 2. Security
+
+Please see [security guidelines](security.md).
+
+## 3. Style and formatting
+
+All workflows and actions must follow consistent formatting rules to improve readability and maintainability.
+Consistency also helps reduce unnecessary diff noise during reviews.
+
+- Format YAML files using [`prettier`](https://prettier.io/) with the default config.
+- Use the `.yml` extension for YAML files (not `.yaml`).
+- Indent using 2 spaces consistently.
+- Add inline comments where logic may not be obvious.
+
+## 4. Workflows
+
+- Repository-specific workflows: `.github/workflows/`
+- Workflow templates for reuse: `.github/workflow-templates/`
```

Comment:
Workflow templates are currently in `workflow-templates`.

Reply:
Good catch - I believe GitHub recommends `.github/workflow-templates/` for repositories, since they will display in the GitHub UI, however I don't believe that applies to this repository as they are examples that should be copied into other repositories. Will fix.